internal/connect/command_cluster_create.go

```diff
@@ -25,6 +25,36 @@ func (c *clusterCommand) newCreateCommand() *cobra.Command {
 		RunE:        c.create,
 		Annotations: map[string]string{pcmd.RunRequirement: pcmd.RequireNonAPIKeyCloudLogin},
 		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: "The configuration file contains configs including offsets for the connector.",
```

Comment:
```suggestion
				Text: "Create a configuration file with connector offsets.",
```

Reply:
Not sure if `Create a configuration file with connector offsets.`  makes sense, we are inputting config file which contains user configs and offsets.

Reply:
ok